section1/reflection.md

```diff
@@ -1,19 +1,41 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+I'm somewhere in-between too long and too short. Sometimes I'll wait to see if someone
+else will ask, and sometimes I just spit something out without really running it through
+my head to see if I can get to the answer.
+It's a battle, but I'm working towards a happy medium the more we go through each session.
```

Comment:
YES, this is pretty common, and being aware of it is really what is most important! You'll more of a groove as you get into full time Turing.